Automattic/sensei
# Fix pattern images urls

### Changes proposed in this Pull Request

* Renames the images files used by the patterns so they use the prefix `patterns-` instead of `pattern-`, fixing an issue where the patterns pointed to the wrong file;

### Testing instructions

1. Create a course;
2. In the Editor Wizard, click on the "Long Sales Page" pattern;
3. Verify that the  image on the block "Deep dive into portrait photography" loads successfully;

### Screenshots

Before:

![image](https://user-images.githubusercontent.com/529864/172261796-0cdf7870-6d9b-4349-9184-da49117d91df.png)

After:

![image](https://user-images.githubusercontent.com/529864/172261839-b2fdab07-4f58-4d56-9e23-46fd0045a4fd.png)
